BookStore/database/migrations/2021_09_24_125304_create_books_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateBooksTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('books', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->string('image');
+            $table->integer('price');
+            $table->string('discreption');
+            $table->integer('category_id');
+            $table->integer('author_id');
```

Comment:
Create foreign key for `author` or `category`
like this:
 `$table->foreignId('author_id');`
`$table->foreignId('category_id');`
 `$table->foreign('category_id')->references('id')->on('categories');`
`$table->foreign('author_id')->references('id')->on('author');`